src/main/java/com/redhat/jenkins/plugins/ci/messaging/topics/TestTopicProvider.java

```diff
@@ -1,37 +0,0 @@
-package com.redhat.jenkins.plugins.ci.messaging.topics;
```

Comment:
Yes, we don't want the TestTopicProvider included. It should never have been checked in.